Formula/ata.rb

```diff
@@ -0,0 +1,19 @@
+class Ata < Formula
+  desc "ChatGPT in the terminal"
+  homepage "https://github.com/rikhuijzer/ata"
+  url "https://github.com/rikhuijzer/ata/archive/refs/tags/v2.0.1.tar.gz"
+  sha256 "6e4a54193f9d875701535f7eaf36225d9b4ec47caf6234827291e6fa6a72951f"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    cd "ata" do
+      system "cargo", "install", *std_cargo_args
+    end
+  end
+
+  test do
+    system "#{bin}/ata", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Okay, done!

It's not the most elegant test, but since this tool doesn't have a built-in command to exit it (you're supposed to just Ctrl-C twice), `pipe.read` would always block forever. So I have to check the output line-by-line with `pipe.gets.chomp`. Still does the trick though.